### ISSUE TITLE
Multi-label output support?

### ISSUE THREAD

**Issue by Alex-Bujorianu** [ahmedfgad/GeneticAlgorithmPython#113]
Does PyGAD support multi-label classification? That is, classification where the classes are not mutually exclusive? My y variable is a numpy array created by the MultiLabelBinarizer. I am trying to build a recommendation algorithm for patients, and the patients typically receive anywhere from 1 to 5 recommendations.

**Comment by ahmedfgad** [comment]
PyGAD supports training Keras and PyTorch models regardless of the problem. So, yes it supports multi-class problems.
